specification/cdn/resource-manager/Microsoft.Cdn/stable/2021-06-01/afdx.json

```diff
@@ -4151,7 +4151,8 @@
             "Rejected",
             "TimedOut",
             "PendingRevalidation",
-            "Approved"
+            "Approved",
+            "RefreshValidationToken",
```

Comment:
Should it be RefreshingValidationToken or Resubmitting? RefreshValidationToken sound more time an action required from the User.

Reply:
Updated to RefreshingValidationToken